tests/schema/__app/generated/schema.graphql

```diff
@@ -33,8 +33,8 @@ input BubbleMembersWhereInput {
 input BubbleWhereInput {
   AND: [BubbleWhereInput!]
   createdAt: DateTimeFilter
-  id: UUIDFilter
-  members: UserFilter
+  id: StringFilter
```

Comment:
BREAKING CHANGE to document

We now [seem to] apply `StringFilter` to model IDs that are of `String` type. This seems sound.

I believe there is an issue about having us generate _not_ `String` type for model IDs but rather GraphQL `ID` types.

When we achieve that, then I would would expect e.g. this to return to something else than `StringFilter`.

For now this actually makes sense and looks more consistent, though.

@Weakky thoughts?

Reply:
I don't think there is really any change here. This diff is just a function of UUID scalar being removed in the first place.

https://github.com/graphql-nexus/nexus-plugin-prisma/pull/805/files#diff-753970daa73034828fbe2f685d2c3f70L4378